src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -85,6 +86,7 @@ internal class GrpcWorkerChannel : IRpcWorkerChannel, IDisposable
         private bool? _cancelCapabilityEnabled;
         private bool _isWorkerApplicationInsightsLoggingEnabled;
         private IHttpProxyService _httpProxyService;
+        private Uri _httpProxyEndpoint;
```

Comment:
Naming is all over the place on this draft right now - internally we have been calling this feature "http proxying" so a lot of variables reflect that including this one.

Reply:
We'll go through a review of those. No need to block at the moment.